server/statistics/hot_peer_cache.go

```diff
@@ -275,17 +269,16 @@ func (f *hotPeerCache) getOldHotPeerStat(regionID, storeID uint64) *HotPeerStat
 
 func (f *hotPeerCache) calcHotThresholds(storeID uint64) []float64 {
 	statKinds := f.kind.RegionStats()
-	mins := make([]float64, len(statKinds))
+	ret := make([]float64, DimLen)
 	for i, k := range statKinds {
```

Comment:
I think we should add a test here

Reply:
no change of length here

Reply:
calcHotThresholds already has unittests.